tools/lsvast/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-option(VAST_ENABLE_LSVAST "Build the lsvast debugging utility" OFF)
```

Comment:
This should stay disabled by default.

Reply:
Yep, just moved it back to its default. This was just when hacking locally, sorry.